R/exp_all_figs_tables.R

```diff
@@ -53,6 +52,8 @@ exp_all_figs_tables <- function(
 
 ) {
 
+  make_rda = TRUE
```

Comment:
Why did you take this out of the arguments?

Reply:
Because if the user is using this function (to export all figs/tables), that implies that all figures and tables will be exported as rda files. So, make_rda would always be true.